src/node/__snapshots__/index.test.ts.snap

```diff
@@ -2,6 +2,6 @@
 
 exports[`node roughUp with a processing instruction returns a roughed up version 1`] = `
 "<?xml-stylesheet href=\\"styles.css\\" type=\\"text/css\\"?><svg viewBox=\\"0 0 100 100\\" xmlns=\\"http://www.w3.org/2000/svg\\">
-  <g><path d=\\"M4.271824857220054 3.2576828692108393 C25.138676139526073 4.068767916318029, 45.42620269954205 3.9021759740076956, 94.80876251123846 5.7751799914985895 M5.613966231234372 5.763947914354503 C26.73993713408709 5.396963874530047, 47.41608370374889 3.874028633069247, 95.38871649373323 4.733868996612728 M96.66412683390081 6.066399225965142 C94.33315986385568 28.624970427714292, 92.36240649214014 49.701983783394105, 94.93674220703542 93.48307580687106 M95.54342797864228 4.811957041732967 C95.43512335112318 34.972813142463565, 94.55234312703833 63.77384696993977, 94.2318487809971 95.57250751089305 M95.92896260879934 96.95408935286105 C74.25662709120661 95.48673654953018, 49.460899468511336 96.1443900601007, 6.234557947143912 96.34666658379138 M95.65541249979287 94.41677750181407 C63.0247921962291 94.13014254430308, 33.23688906338065 94.73722935179248, 4.4081691252067685 94.73184285592288 M5.455919584259391 94.6942517850548 C6.1764527195133265 70.67311355192214, 5.3274377816356715 46.79456298053264, 3.3789065089076757 5.196091482415795 M5.206786249764264 94.77906237076968 C3.8285125856287774 60.11709273606538, 3.804144150484353 27.934595261700437, 4.671614293940365 5.493582795374095\\" stroke=\\"#000\\" stroke-width=\\"1\\" fill=\\"none\\"/></g>
+  <g><path d=\\"M6.462354740127921 6.325660714879632 C34.251420262269676 2.48141179298982, 65.85282365977764 4.60687498068437, 93.5202522482723 4.096276352182031 M4.46973726619035 4.687576274387538 C34.59542376548052 5.113023323845118, 61.87379447650165 4.477553559374064, 95.91062808502465 5.928292225115001 M93.76622631214559 5.510313579812646 C96.25938451578841 28.266343734227124, 94.46523446133361 50.19505188241602, 93.09794851206243 93.07262576557696 M95.23880911339074 4.554712484590709 C94.308279365208 30.868172479793433, 94.51635092487558 56.42291493806989, 94.88826182950288 95.28677193354815 M96.84552474878728 96.32514871098101 C59.34138903860003 94.88877208093182, 27.062555070966482 95.31565357306972, 6.883139511570334 96.02736301161349 M94.11932829488069 94.09612218569964 C69.63432473130524 95.55104413786903, 42.084781258367 94.99353343406692, 5.045000378973782 95.21329344343394 M6.3339780773967505 95.45577401854098 C5.325689192954451 69.09420007187873, 6.209307845775038 44.25991837680339, 3.359162649139762 4.140236625447869 M4.355465953238308 94.69702876638621 C4.636683759558946 75.00010431557894, 5.785503533948213 56.049772386439145, 5.340835188515484 5.45538483094424\\" stroke=\\"#000\\" stroke-width=\\"1\\" fill=\\"none\\"/></g>
```

Comment:
Wasn't expecting this to change. Adding the new `describe.each` seems to cause a different seed to be generated.

Reply:
Weird, I thought I was mocking `Math.random` successfully to avoid this kind of churn: https://github.com/unindented/rougher/blob/master/config/jest/random.ts